napari/_qt/qt_viewer.py

```diff
@@ -1072,7 +1073,8 @@ def dropEvent(self, event):
         filenames = []
         for url in event.mimeData().urls():
             if url.isLocalFile():
-                filenames.append(url.toLocalFile())
+                # directories get a trailing "/", Path conversion removes it
+                filenames.append(str(Path(url.toLocalFile())))
```

Comment:
I feel like `url.toLocalFile().rstrip('/')` would fix this but this is fine :joy:

Reply:
I know I was just paranoid for some reason...